index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+
+
+    
+    <script  src="js/index.js"></script>
+
+    <header>
+      <div class="head1">
+        <center>
+          <h1>Dr. Norman Borlaug</h1>
```

Comment:
- [ ] The logo text suppose to be wrap by a link `a` tag. This is for three reasons: convention, semantics, and user experience(UX). ℹ️